addons/website/views/website_pages_views.xml

```diff
@@ -133,9 +133,9 @@
                             title="Home page of the current website"/>
                         <field name="name" />
                     </div>
-                    <div t-if="record.website_id.value" class="text-truncate fw-bold text-muted">
+                    <div t-if="record.website_id.value" class="text-truncate fw-bold text-muted" groups="website.group_multi_website">
                         <i class="fa fa-globe me-1" title="Website"/>
-                        <field name="website_id" groups="website.group_multi_website"/>
```

Comment:
No attribute removal in stable version please: an xpath can be easily based on it:
1. Add the groups on the parent in stable, do not remove this one
2. In the master forward-port, then you'll remove the useless one

Reply:
> 1. Add the groups on the parent in stable, do not remove this one

done 👍🙂